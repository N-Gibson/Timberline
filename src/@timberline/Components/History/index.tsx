import React from 'react'
import Hallway from '../../assets/Venado_Hallway.jpg'
import './History.scss'

const History = () => {
  return (
    <section className="history">
      <article className="history_article">
        <p className="history_article_text">
          Timberline Inc. was founded in 1995 with the goals of creating quality
          solutions in the housing market. It has evolved into a company that
          can do a bit of everything. Etc... this narrative will need some work.
        </p>
        <img
          className="history_main_image"
          src={Hallway}
          alt="hallway in house"
        />
      </article>
    </section>
  )
}

export { History }
