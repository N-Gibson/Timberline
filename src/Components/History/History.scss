.history {
  height: 75vh;
  display: flex;
  align-items: center;
}

.history_article {
  display: grid;
  grid-template-columns: 2fr 1fr;
  grid-template-rows: auto;
  justify-items: center;
  align-items: center;
  grid-gap: 2em;
  padding: 2em;
}

.history_article_text {
  background-color: #c8ad96;
  padding: 2em;
  border-radius: 6px;
  line-height: 2;
  font-size: 1.25em;
  box-shadow: 10px 0px 20px 0px #00000040;
}

.history_main_image {
  width: 100%;
  border-radius: 6px;
  box-shadow: 10px 0px 20px 0px #00000040;
}
