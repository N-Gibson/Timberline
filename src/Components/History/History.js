import React from "react";
import Hallway from "../../assets/Venado_Hallway.jpg";
import "./History.scss";

export default function History() {
  return (
    <section className="about">
      <article className="about_article">
        <p className="about_article_text">
          Timberline Inc. was founded in 1995 with the goals of creating quality
          solutions in the housing market. It has evolved into a company that
          can do a bit of everything. Etc... this narrative will need some work.
        </p>
        <img
          className="about_main_image"
          src={Hallway}
          alt="(change later when pic is available)"
        />
      </article>
    </section>
  );
}
