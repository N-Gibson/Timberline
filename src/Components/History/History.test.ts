import React from "react";
import { shallow } from "enzyme";
import History from "./History";

describe("History", () => {
  const wrapper = shallow(<History />);

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
