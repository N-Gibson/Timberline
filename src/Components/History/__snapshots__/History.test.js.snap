// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History should match snapshot 1`] = `
<section
  className="about"
>
  <article
    className="about_article"
  >
    <p
      className="about_article_text"
    >
      Timberline Inc. was founded in 1995 with the goals of creating quality solutions in the housing market. It has evolved into a company that can do a bit of everything. Etc... this narrative will need some work.
    </p>
    <img
      alt="hallway in house"
      className="about_main_image"
      src="Venado_Hallway.jpg"
    />
  </article>
</section>
`;
